File: Service/Billing/RMA.Service.Billing.Contracts/Entities/InterestCalculationRequest.cs

using RMA.Service.Admin.MasterDataManager.Contracts.Enums;

namespace RMA.Service.Admin.MasterDataManager.Contracts.Entities
{
    public class InterestCalculationRequest
    {
        public IndustryClassEnum? IndustryClass { get; set; }
        public int? PeriodId { get; set; }
        public int? RolePlayerId { get; set; }
        public int? PolicyId { get; set; }
        public int? ProductCategoryId { get; set; }
    }
}

File: Web/RMA-Web-Shell/projects/fincare/src/app/billing-manager/models/interest-calculation-request.ts
import { IndustryClassEnum } from "projects/shared-models-lib/src/lib/enums/industry-class.enum";

export class InterestCalculationRequest {
        industryClass: IndustryClassEnum;
        periodId: number;
        rolePlayerId: number;
        policyId: number;
        productCategoryId: number;
}



